charts/kspm-collector/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kspm-collector
 description: Sysdig KSPM collector
 
-version: 0.3.4
-appVersion: 1.26.0
+version: 0.3.5
```

Comment:
I expect the bump being a minor, not a patch release since we are bumping a minor in the image we are using.